test/infer_rt_test.py

```diff
@@ -193,3 +194,41 @@ def test_smoothing_and_causality():
         ),
         "test_smoothing_and_causality",
     )
+
+
+def test_generate_infection_rate_metric_no_region_given():
+    FIPS = []
+    REGIONS = [infer_rt.RegionalInput.from_fips(region) for region in FIPS]
+    df = cli._generate_infection_rate_metric(regions=REGIONS)
```

Comment:
i'd prefer not to call this with the variable name, since it's a positional/non-optional parameter.  
i.e. `cli._generate_infection_rate_metric(REGIONS)`.  main reason for preferring this is 2 parts:
 - makes it clear which arguments are actually optional
 - lets you rename the positional argument without having to update everywhere

Reply:
Fixed.